ev3dev2/display.py

```diff
@@ -347,7 +347,7 @@ def rectangle(self, clear_screen=True, x=10, y=10, width=80, height=40, fill_col
         if clear_screen:
             self.clear()
 
-        return self.draw.rectangle((x, y, width, height), fill=fill_color, outline=outline_color)
+        return self.draw.rectangle((x, y, x+width, y+height), fill=fill_color, outline=outline_color)
```

Comment:
So we're calling this a code bug rather than a documentation issue? I don't have anything against that, but it's worth keeping in mind that this will silently change code behavior.

Reply:
yeah I looked at the PIL docs...it is a code bug on our part.

Reply:
True...it would probably be less confusing to stay consistent with PIL long term though. Our API matches there everywhere else (I think)

Reply:
Oh, really? Maybe I am looking in the wrong place. It seems to me that PIL's methods use point pairs: https://pillow.readthedocs.io/en/3.1.x/reference/Image.html

Reply:
True, I see what you are saying now.  Given that our ``rectangle()`` accepts ``x, y, width, height`` it seems like a less drastic change to fix the behavior (ie the code change in the current pull request)  than it does to change our ``rectange()`` to accept ``x1, y1, x2, y2``.

Reply:
Agreed; I think the naming/docs change would make the most sense given that we've released it with that behavior already.